.gitignore

```diff
@@ -1,4 +1,7 @@
 node_modules
 .DS_Store
 yarn-error.log
-dist
\ No newline at end of file
+dist
+
+# testing
+/coverage
```

Comment:
I think you want a trailing slash not a leading slash.  Also don't forget to `git rm -rf coverage/` in the root